js/views/chatHeadsVw.js

```diff
@@ -18,6 +18,18 @@ module.exports = baseVw.extend({
         this.createChatHead(md).render().el
       );
     });
+
+    this.$chatHeadsContainer = $('.chatConversationHeads');
+    this.$headContainer = $('<div />');
+
+    this.showPerScroll = 10;
```

Comment:
I'm pretty sure we're going to need more than 10 here. The tricky thing is that the chat view spans the total height of the app, which is variable (e.g. the user can go into fullscreen mode). If we don't show enough to take up the full height of the app, there will never be a possibility for the user to scroll and lazy load the rest.